web/src/js/__tests__/components/ContentView/__snapshots__/CodeEditorSpec.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CodeEditor Component should render correctly 1`] = `
+<div
```

Comment:
`CodeMirror` itself is too complex, it use `React.findDOMNode` and `ref` tricks to implement its logic. Unfortunately, `react-test-render` has poor support for these two statements. It will throw some strange exceptions.
The only way that came up to me is mock the whole `CodeMirror`, as a dependency, it should have been fully tested, right? Or we could turn to `TestUtils.renderIntoDocument` to do DOM test, but we need to mock many other functions like `document.createRange` for `CodeMirror`, basically doing another render test for `CodeMirror` component.
Do you have any idea about how to deal with this `CodeMirror` ?
BTW, I don't understand why the codecov coverge dropped.